core/db/migrate/20221229132350_create_spree_google_export_options.rb

```diff
@@ -0,0 +1,16 @@
+class CreateSpreeGoogleExportOptions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :spree_google_export_options do |t|
+      t.belongs_to :spree_store, foreign_key: true
+
+      keys = [:material, :brand, :gender, :condition, :gtin, :mpn, :adult, :multipack, :is_bundle, :color, :pattern,
```

Comment:
Are these keys used anywhere right now, or is it a WIP concept, to assign these details to products?

Reply:
They are WIP concept, I wasn't sure where ths sort of information about products would be stored, so for now I put it product properties.

Reply:
@rafalcymerys